10485.xml

```diff
@@ -0,0 +1,191 @@
+				<Name>Age </Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>s</Units>
+				<Description><![CDATA[The age is a measure of how long it has been since the last communication or interaction with that particular network neighbor.]]></Description>
+			</Item>
+			<Item ID="3">
+				<Name>Average RSSI</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Multiple</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>dBm</Units>
+				<Description><![CDATA[The average Received Signal Strength Indication (RSSI) over a specific period.]]></Description>
+			</Item>
+			<Item ID="4">
+				<Name>Last RSSI</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>dBm</Units>
+				<Description><![CDATA[The instantaneous Received Signal Strength Indication (RSSI).]]></Description>
+			</Item>
+			<Item ID="5">
+				<Name>Rx On When Idle</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[Neighbor mode R represents "RX on when idle".]]></Description>
+			</Item>
+			<Item ID="6">
+				<Name>Full Thread Device</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[Neighbor mode D represents "Full Thread device".]]></Description>
+			</Item>
+			<Item ID="7">
+				<Name>Full Network Data</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[Neighbor mode N represents "Full network data".]]></Description>
+			</Item>
+			<Item ID="8">
+				<Name>Extended MAC Address </Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[ Extended Medium Access Control (MAC).]]></Description>
+			</Item>
+			<Item ID="9">
+				<Name>Link Quality In</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>Link Quality is an integer in [0, 3]. A higher link quality indicates a more usable link, with 0 indicating that the
```

Comment:
Range enumeration can't have free text. Seems "0..3" is appropriate here. Rest should be in description.